web/src/routes/(user)/map/[[photos=photos]]/[[assetId=id]]/+page.svelte

```diff
@@ -130,12 +186,97 @@
     await navigate({ targetRoute: 'current', assetId: $viewingAsset.id });
     return asset;
   }
+
+  function onChangeBounds(bounds: maplibregl.LngLatBounds) {
+    let lng_e: number, lng_w: number, lat_e: number, lat_w: number;
+
+    /*
+    /*
+    /* longitude and latitude can be >180 and <180 with maplibre
+    /* that part fixes it to always have longitude coordinates -180 < x1, x2 < 180
+    /*
+    */
+    if (Math.abs(bounds._ne.lng) + Math.abs(bounds._sw.lng) > 360) {
+      lng_e = -180;
+      lng_w = 180;
+    } else if (Math.abs(bounds._sw.lng) > 180) {
+      lng_e = bounds._sw.lng + 360;
```

Comment:
Where did you get from that you need to add 360 if `|lng| > 180`? Is that per the maplibre docs?